docs/players.rst

```diff
@@ -95,7 +95,15 @@ The stable version of VLC cannot play Youtube Live streams. This a known bug
 that is fixed in VLC 3, which is still in development. You can also try
 using a different player.
 
+Youtube Live does not work with Mplayer
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Some versions of Mplayer cannot play Youtube Live streams. And errors like:
+Cannot seek backward in linear streams!
+Seek failed
+Switching to a recent fork such as mpv resolves the issue.
+
 When using OMXPlayer the stream stops unexpectedly
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 When reading from a fifo pipe OMXPlayer will quit when there is no data, to fix
 this you can supply the timeout option to OMXPlayer using :option:`--player "omxplayer --timeout 20" <--player>`.
+For live streams it might be beneficial to also add :option:`--live`.
```

Comment:
https://travis-ci.org/streamlink/streamlink/jobs/452411724#L1174

The `:option:` directive is only meant for Streamlink parameters:
https://github.com/streamlink/streamlink/blob/master/docs/ext_argparse.py#L82

Use this instead for regular parameters:
```
``--live``
```